proxy.go

```diff
@@ -119,13 +124,14 @@ func unixAddr(uri string) (string, error) {
 
 func logger() *logrus.Entry {
 	return proxyLog.WithFields(logrus.Fields{
-		"name":   proxyName,
-		"pid":    os.Getpid(),
-		"source": "proxy",
+		"name":    proxyName,
+		"pid":     os.Getpid(),
+		"source":  "proxy",
+		"sandbox": sandboxID,
```

Comment:
since `sandboxID` is optional, I think you should add it only if it's not empty, right?

Reply:
I guess that's fair :) Branch updated.